src/ReactTinyLink.tsx

```diff
@@ -29,10 +29,15 @@ export const ReactTinyLink: React.FC<IReactTinyLinkProps> = ({
   const defaultMediaArr = defaultMedia ? [defaultMedia] : []
   const [data, loading] = useMountFetch(
     url, proxyUrl, defaultMediaArr, defaultData(url, defaultMediaArr), onError, onSuccess)
+  const handleClick = (e) => {
+    if (!onClick) return
+    e.preventDefault()
+    onClick(e, data)
+  }
 
   return (
     <>
-      {!onClick && <Card className="react_tinylink_card" cardSize={cardSize} href={url} width={width} isShownGraphic={showGraphic}>
+      {<Card onClick={handleClick} className="react_tinylink_card" cardSize={cardSize} href={url} width={width} isShownGraphic={showGraphic}>
```

Comment:
Shall we put props implementation like this ? Since the handling flow is straight forward.
```tsx
{<Card onClick={(e)=>{ if (!onClick) return
    e.preventDefault()
    onClick(e, data)}} className="react_tinylink_card" cardSize={cardSize} href={url} width={width} isShownGraphic={showGraphic}>
```